documentation/modules/overview/con-key-features-kafka.adoc

```diff
@@ -0,0 +1,17 @@
+// This module is included in:
+//
+// overview/assembly-key-features.adoc
+
+[id="key-features-kafka_{context}"]
+= Kafka capabilities
```

Comment:
= Kafka use cases

Kafka's capabilities make it suitable for:

* Stream processing so that applications can respond to data in real time
* Integration of microservices through an event-driven publish-subscribe architecture
* Streaming of analytics and activity tracking through metrics and log collection

Reply:
I think we should really just mirror this here: http://kafka.apache.org/uses (I mean the headings as bullet points, not the whole content). I think we should also add "Event driven architectures" to that since that is in right now.

Reply:
Kafka's capabilities make it suitable for:

* Event-driven architectures
* Event sourcing to capture changes to the state of an application as a log of events
* Message brokering
* Website activity tracking
* Operational monitoring through metrics
* Log collection and aggregation
* Commit logs for distributed systems 
* Stream processing so that applications can respond to data in real time